API.md

```diff
@@ -0,0 +1,49 @@
+<pre>Activities</pre>
+title | CharField  
+subtitle | CharField  
+description | TextField  
+image_url | UrlField  
+image_retina_url | UrlField  
+url | UrlField  
+difficulty | CharField (choices)  
+duration | CharField (choices)  
+**Skills** | *:*  
+**WebLitSkills** | *:*  
+**Topic** | 1:*  
+**LessonPlan** | *:*  
+**locale** | CharField (choices) (pycountry)  
+translation_of | ‘self’  
+
+
+
+<pre>Skills</pre>
```

Comment:
@hannahkane probably not necessary here (although a comment in the code would be a good idea). These names are what the database/code know of. What is shown to the user via the API or via the Admin interface can be customized to whatever we want.